tasks/release.py

```diff
@@ -1525,3 +1525,21 @@ def _create_build_links_patterns(current_version, new_version):
     patterns[current_minor_version.replace("-rc", "~rc")] = new_minor_version.replace("-rc", "~rc")
 
     return patterns
+
+
+@task
+def get_active_release_branch(_ctx):
+    """
+    Determine what is the current active release branch for the Agent.
+    If release started and code freeze is in place - main branch is considered active.
+    If release started and code freeze is over - release branch is considered active.
+    """
+    gh = GithubAPI('datadog/datadog-agent')
+    latest_release = gh.latest_release()
+    version = _create_version_from_match(VERSION_RE.search(latest_release))
+    next_version = version.next_version(bump_minor=True)
+    release_branch = gh.get_branch(next_version.branch())
```

Comment:
I agree that as is it shouldn't change much, I was more worried about if/when we start using this task in more locations.
As for git's concept of release, sure, but AFAICS you're checking the existence of a branch which is easily done through git itself.

That being said, it definitely isn't a blocker as far as I'm concerned

Reply:
I am checking existing of a branch that I determine based on the latest release name. This can be done again by pulling all branches and looking for the "newest" one but there are more cases to cover.
I will keep the rate limit aspect in mind if we start to use it more than assumed.